hypha/apply/projects/templates/application_projects/includes/deliverables_block.html

```diff
@@ -0,0 +1,39 @@
+<div class="sidebar__inner">
+
+    <h5>Choose deliverables</h5>
+    <form id="add-deliverables", action="" data-projectid="{{ project.id }}" data-invoiceid="{{ invoice.id }}">
+        <div class="select-deliverables">
+            <select name="deliverables" id="deliverables">
+            <option value="deliverables">Invoice Covers Deliverables</option>
+            {% for deliverable in deliverables %}
+                <option id="{{ deliverable.id }}" value="{{ deliverable.id }}" data-deliverableid="{{ deliverable.id }}" data-availabletoinvoice="{{ deliverable.available_to_invoice }}">{{ deliverable.name }} (${{ deliverable.unit_price }})</option>
+            {% endfor %}
+            </select>
+        </div>
+        <br>
+        <div class="available-to-invoice">
+            <b>Available to invoice: </b>
+        </div>
+        <br>
+        <div class="quantity">
+            <b><label for="quantity">Quantity:</label></b>
+            <input type="number" id="quantity" name="quantity" min="1">
+        </div>
+        <br>
+        <input type="submit" value="Add Deliverable">
+    </form>
+    <br>
+    <div id="list-deliverables">
+        <div class="deliverables">
+            {% for deliverable in invoice.deliverables.all %}
+                <b>{{ deliverable.deliverable.name }} ({{ deliverable.quantity }} ${{deliverable.deliverable.unit_price}})</b><a href="{% url "api:v1:remove-deliverables" pk=deliverable.pk invoice_pk=invoice.pk project_pk=project.pk %}"> Remove</a>
+            {% endfor %}
+        </div>
+        {% if invoice.deliverables_total_amount.total %}
+            <div class="total">
+                <b>...</b><br>
+                <b>Total: ${{ invoice.deliverables_total_amount.total }}</b>
```

Comment:
Instead of hardcoding the currency symbol please use `settings.CURRENCY_SYMBOL` or in template like this `{{ CURRENCY_SYMBOL }}`.

https://github.com/HyphaApp/hypha/blob/637b6b9ee3d4cb2460cc9e8f6ecd84afbb86ecdc/hypha/settings/base.py#L333

https://github.com/HyphaApp/hypha/blob/637b6b9ee3d4cb2460cc9e8f6ecd84afbb86ecdc/hypha/public/utils/context_processors.py#L8-L20